src/components/ProblemsTable.tsx

```diff
@@ -244,6 +284,27 @@ export const ProblemsTable = ({
 
   return (
     <Table
+      onRowClick={(event: React.MouseEvent<Element, MouseEvent>, row: Row<TableRow | TableRow['subRows']>) => {
+        const element = event.target as HTMLTableRowElement;
+        element.parentElement?.classList.add('active');
+        context?.setCurrentAuditInfo({
+          ...row.original,
+          fileName: row.original.filename ?? getFileNameFromId(row.original.fileId),
+        });
+
+        // event listener added as component does not have an 'outside clicked' handler
```

Comment:
please mention if any suggestions for more suiting implementation, in terms of React or component library